### PR TITLE
PAYARA-1509 condition for access logger may be read in from domain.xml

### LINKED ISSUE #1450: Support conditional Access Logs
# Description #
----------

Support condition on the Access Log in Payara. See https://tomcat.apache.org/tomcat-7.0-doc/config/valve.html#Access_Log_Valve/Attributes for the equivalent in Tomcat. The source code in PEAccessLog supports the conditrion attribute but there is no way to configure it in the domain.xml
See https://github.com/payara/Payara/blob/master/appserver/web/web-glue/src/main/java/com/sun/enterprise/web/PEAccessLogValve.java#L593 

## Expected Outcome

Conditional access logging is supported.

## Current Outcome

This is a pretty simple fix and will extend the functionality of access logging to bring it into line with the Tomcat equivalent.

### PR DESCRIPTION
**Pull request by Cousjava**
fixes #1450

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins, would you kindly?

File: nucleus/admin/config-api/src/main/java/com/sun/enterprise/config/serverbeans/ConfigBeansUtilities.java [hunk 1]
```diff
@@ -74,6 +75,13 @@ public ConfigBeansUtilities(Applications paramApps, Domain paramDomain) {
         domain = paramDomain;
     }
 
+    /*
```
Comment (smillidge) [inline nucleus/admin/config-api/src/main/java/com/sun/enterprise/config/serverbeans/ConfigBeansUtilities.java]: Why is this change needed if the default if not available will be null anyway?

File: appserver/web/web-glue/src/main/java/com/sun/enterprise/web/PEAccessLogValve.java [hunk 2]
```diff
@@ -908,6 +908,14 @@ void updateAccessLogAttributes(HttpService httpService,
             historyFiles = new LinkedList<File>();
         }
         
+        //Conditional access log
+        if (accessLogConfig != null) {
```
Comment (smillidge) [inline appserver/web/web-glue/src/main/java/com/sun/enterprise/web/PEAccessLogValve.java]: Why check if not null and then do an else that sets it to null. Why not just set it regardless and not bother with the getDefaultCondition change?

File: nucleus/admin/config-api/src/main/java/com/sun/enterprise/config/serverbeans/ConfigBeansUtilities.java (hunk 1 above)
Comment (Cousjava) [inline nucleus/admin/config-api/src/main/java/com/sun/enterprise/config/serverbeans/ConfigBeansUtilities.java]: I wrote it this way to match how others are set, and I'm not sure if the default condition is meant to be null - the other defaults mention that they come from a dtd which I have not been able to find.
